ppr-ui/src/views/exemption/Exemptions.vue

```diff
@@ -133,8 +142,28 @@ export default defineComponent({
     }
 
     const submit = async (): Promise<void> => {
+      localState.loading = true
+      localState.validate = true
+      await nextTick()
+      await scrollToFirstVisibleErrorComponent()
+
+      if (isMhrExemptionValid.value) {
+        // Construct payload
+        const payload = buildExemptionPayload()
```

Comment:
I like the _build payload_ naming, and will rename the _build api data_ functions. 👍🏻

Reply:
Right on!